custom/icds_reports/utils/data_pull.sql

```diff
@@ -0,0 +1,50 @@
+CREATE TEMPORARY TABLE "temp_visit_table" AS (
+    SELECT
+        location_id as supervisor_id,
+        COUNT (DISTINCT awc_id) as unique_visits
+    FROM "ucr_icds-cas_static-awc_mgt_forms_ad1b11f0"
+    WHERE
+        submitted_on>= '2020-02-01'
+        AND submitted_on<'2020-04-01'
+        AND location_entered IS NOT NULL
+        AND location_entered <> ''
+    GROUP BY location_id
```

Comment:
should it be supervisor_id as you are aliasing it?

Reply:
nope